.changeset/tricky-cobras-provide.md

```diff
@@ -0,0 +1,11 @@
+---
+"@astrojs/tailwind": patch
+"@astrojs/markdoc": patch
+"@astrojs/svelte": patch
+"@astrojs/react": patch
+"@astrojs/mdx": patch
+"@astrojs/vue": patch
+"astro": patch
```

Comment:
```suggestion
"astro": patch
```

We only need the patch for `astro` as the rest has `vite` only as a dev dependency